src/commands/postDeploy/checkLinuxWebAppDownDetector.ts

```diff
@@ -50,26 +41,33 @@ export async function checkLinuxWebAppDownDetector(correlationId: string, node:
             }
 
             if (Date.now() > detectorTimeoutMs) {
-                if (showOutput) {
-                    const noIssuesFound: string = `Diagnosing for "${node.root.client.siteName}" has timed out.`;
-                    ext.outputChannel.appendLog(noIssuesFound);
-                    context.telemetry.properties.timedOut = 'true';
-                }
+                context.telemetry.properties.timedOut = 'true';
                 return undefined;
             }
 
-            // tslint:disable-next-line: no-unsafe-any
-            detectorErrorMessage = await getLinuxDetectorError(context, linuxContainerStartFailureId, node, deployResultTime, node.root.client.fullName);
+            // time constraint of being less than the current time by 14.5 minutes is enforced system wide for all detectors
+            // however LinuxLogViewer detector ignores it and gets the most recent data
+            const nowTime: number = Date.now();
+            const startTimeDate: Date = new Date(nowTime - (60 * 60 * 1000));
+            const endTimeDate: Date = new Date(nowTime - (30 * 60 * 1000));
+
+            const startTime: string = moment.utc(startTimeDate).format('YYYY-MM-DDTHH:mm');
+            const endTime: string = moment.utc(endTimeDate).format('YYYY-MM-DDTHH:mm');
+
+            detectorErrorMessage = await getLinuxDetectorError(context, linuxLogViewer, node, startTime, endTime, deployment.endTime);
 
             if (!detectorErrorMessage) {
-                // poll every minute
-                await delay(1000 * 60);
+                // poll every 10 seconds
+                await delay(1000 * 10);
             }
         }
 
+        const enableDetectorsSetting: string = 'enableDetectors';
+        const showOutput: boolean | undefined = getWorkspaceSetting<boolean>(enableDetectorsSetting);
         if (showOutput) {
```

Comment:
I was thinking it could be time to remove this flag.

Reply:
Personally I'm still waiting to see examples of common, real-life errors that this finds and displays to the user in an easy-to-understand, actionable way

Reply:
Okay demos in standup have satisfied me - 🚫🚩

Reply:
Emoji game ✊